File: strings/python/cpf.py
import re


def remove_non_digits(string):
    char_array = []

    for char in string:
        if char.isdigit():
            char_array.append(char)

    return ''.join(char_array)


def remove_non_digits2(string):
    regex = r'\D'
    return re.sub(regex, '', string)


print(remove_non_digits("949.237.847-99"))
print(remove_non_digits2("949.237.847-99"))

File: strings/javascript/cpf.js
function removeNonDigits(str) {
    let charArray = []; 

    for (let i = 0; i < str.length; i++) {
        if (str[i] >= '0' && str[i] <= '9') {
            charArray.push(str[i]);
        }
    }

    return charArray.join("");
}


function removeNonDigits2(str) {
    const regex = /\D/g;
    return str.replace(regex, "");
}


console.log(removeNonDigits("949.237.847-99"));
console.log(removeNonDigits2("949.237.847-99"));
